ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -431,7 +425,8 @@ double UncoupledProcessesTimeLoop::computeTimeStepping(
         }
         else
         {
-            if (t < _end_time)
+            if (t < _end_time || std::abs(t - _end_time) <
+                                     std::numeric_limits<double>::epsilon())
```

Comment:
Just find a std function: std::islessequal.

Reply:
Thanks, I didn't know that function. But from [cppreference](https://en.cppreference.com/w/cpp/numeric/math/islessequal) it seems that the only difference to `operator<=(x, y)` is that the floating point exceptions are not raised (and we do like exceptions).

Reply:
I see.